docs/codehealth_rotation.md

```diff
@@ -0,0 +1,87 @@
+# Code Health
+
+## Overview
+
+This is an outline for the CodeHealth rotation, which aims to proactively manage
+technical debt and maintain our codebase by keeping it up-to-date with upstream
```

Comment:
Is this specifically to align with Chromium? Or would knocking out `brave-core` issues marked as `dev-concern` be under scope too?

Reply:
Actually, what is the difference between the `code-health` label and `dev-concern`? I'm guessing `code-health` is a strict subset of `dev-concern`?

Reply:
We are marking them as `code-health` but I guess they also fall under `dev-concern`. cc @bsclifton

Reply:
`dev-concern` is an older label, but still in use, for things that are refactors or not using best practices. Things that would have no visible difference to a product person.

Reply:
I just noted your other comment on this now. I'm gonna remove references to `dev-concern`.

Reply:
After some discussion with Bondy, I have removed any references to `dev-concern` as it is being retired.